modules/ROOT/pages/amazon/amazon-dynamodb-connector-examples.adoc

```diff
@@ -0,0 +1,700 @@
+			<dynamodb:attribute-definition attributeName="DueDate" attributeType="STRING" />
+		</dynamodb:attribute-definitions>
+		<dynamodb:key-schemas>
+			<dynamodb:key-schema-element attributeName="IssueId" keyType="HASH" />
+			<dynamodb:key-schema-element attributeName="Title" keyType="RANGE" />
+		</dynamodb:key-schemas>
+	</dynamodb:create-table>
+		<logger level="INFO" doc:name="Logger"
+    message="#[payload]"/>
+</flow>
+</mule>
+----
+
+=== Run the Project
+
+Save and run the project as a Mule app:
+
+. In *Package Explorer*, right-click the project and click *Run As* > *Mule Application*.
+. Open a browser and check the response after you enter the `+http://localhost:8081/createtable+` URL.
++
+You should see the generated response from the `Create table` operation in the Mule console.
+
+[[delete-table]]
+== Delete an Amazon DynamoDB Table
+
+This example shows how to use Amazon DynamoDB connector to delete an Amazon DynamoDB table.
+
+=== Flow for Deleting a DynamoDB Table
+
+The following screenshot shows an Anypoint Studio flow for deleting an Amazon DynamoDB table:
+
+.Use this sample flow to delete an Amazon DynamoDB table.
+image::amazon/amazon-dynamodb-delete-table-flow.png[Studio flow for deleting an Amazon DynamoDB table]
+
+=== Preliminary Steps
+
+Follow these steps to start deleting an Amazon DynamoDB table:
+
+. Create a new Mule project in Studio.
+. Add the following properties to the `mule-artifact.properties` file to hold your Amazon DynamoDB credentials:
++
+[source,text,linenums]
+----
+config.accesskey=<Access Key>
+config.secretkey=<Secret Key>
+----
++
+. Place the `mule-artifact.properties` file in the project's `src/main/resources` directory.
+
+=== Configure HTTP Listener
+
+Configure `HTTP Listener` to listen for HTTP requests on the `/deletetable` path:
+
+. In the *Mule Palette*, search for `http` and select the *Listener* operation.
+. Drag the *Listener* operation onto the Studio canvas.
+. In the Listener properties window, click the plus sign (+) next to the *Connector configuration* field to access the global element configuration fields for `HTTP Listener`.
+. In the *Host* field, select `localhost` and click *OK*.
+. In the *Path* field on the Listener properties window, enter `/deletetable`.
+
+=== Configure the Delete Table Operation
+
+Configure the `Delete table` operation to delete the `Issues` table.
+
+. In the *Mule Palette*, click *Search in Exchange* and search for `amazon dynamodb`.
+. Select *Amazon DynamoDB Connector*, click *Add>*, and then click *Finish*.
+. Drag the *Delete Table* operation to the right of *Listener* on the Studio canvas.
+. Click the *Global Elements* link below the flow.
+. Add the properties file you created earlier to the *Configuration properties* field, as described in <<introduction/intro-connector-configuration-overview.adoc#property-placeholders,Use Property Placeholders for Property Values>>.
```

Comment:
ditto re. sending the user out to another topic; omit or move to See Also or prerequisties